doc/guides/west/extensions.rst

```diff
@@ -0,0 +1,244 @@
+.. _west-extensions:
```

Comment:
I've got no idea why this .rst file is flagged as missing a license and copyright, unless the python code block on line 77 is somehow noticed?

Reply:
It's scancode doing that. We can safely ignore it.